lib/screens/v2/candidate_generator/widgets/departures.ex

```diff
@@ -139,6 +139,8 @@ defmodule Screens.V2.CandidateGenerator.Widgets.Departures do
 
   defp normal_section_rows({:ok, departures}), do: departures
 
+  defp normal_section_rows(:wayfinding_only), do: []
```

Comment:
suggestion: I think this extra clause would not be needed if we returned `{:ok, []}` instead of using the separate value `:wayfinding_only` for this purpose.